File: packages/libbird/android/src/main/java/dev/flutterbird/ladybird/RequestServerService.java
package dev.flutterbird.ladybird;

import android.os.Message;

public final class RequestServerService extends LadybirdServiceBase {
    public RequestServerService() {
        super("RequestServerService");
    }

    @Override
    boolean handleServiceSpecificMessage(Message msg) {
        return false;
    }

    static {
        System.loadLibrary("requestserverservice");
    }
}

File: packages/libbird/android/src/main/kotlin/org/serenityos/ladybird/RequestServerService.kt
package org.serenityos.ladybird

import android.os.Message

class RequestServerService : LadybirdServiceBase("RequestServerService") {
  override fun handleServiceSpecificMessage(msg: Message): Boolean = false

  companion object {
    init {
      System.loadLibrary("requestserverservice")
    }
  }
}
